packages/gestalt/src/index.js

```diff
@@ -34,8 +34,6 @@ import Letterbox from './Letterbox.js';
 import Link from './Link.js';
 import Mask from './Mask.js';
 import Masonry from './Masonry.js';
-import MasonryDefaultLayout from './layouts/MasonryLayout.js';
```

Comment:
Please see https://phabricator.pinadmin.com/D906449 where we remove these imports from pinboard/webapp